src/link/object.c

```diff
@@ -331,9 +338,8 @@ obj_ReadOpenFile(FILE * pObjfile, char *tzObjectfile)
 	tzHeader[4] = 0;
 	if (strncmp(tzHeader, "RGB", 3) == 0) {
 		switch (tzHeader[3]) {
-		case '3':
-		case '4': // V4 supports OAM sections, but is otherwise identical
-			obj_ReadRGB(pObjfile);
+		case '5':
```

Comment:
Ok, then. It probably makes sense to have the whole string somewhere, not just the number.